Purchases/Purchasing/Purchases.swift

```diff
@@ -776,7 +776,15 @@ public extension Purchases {
     ///
     /// - Seealso `PurchasesDelegate.purchases(_ purchases: Purchases, didReceiveUpdated:)`
     /// - Seealso `Purchases.customerInfo()`
-    /// - Note: this method is not thread-safe.
+    ///  #### Example:
+    /// ```swift
+    ///     for await customerInfo in Purchases.shared.customerInfoStream {
+    ///       // this gets called whenever new CustomerInfo is available
+    ///       let entitlements = customerInfo.entitlements
+    ///       ...
+    ///     }
+    /// ```
```

Comment:
this looks like this in docs 
<img width="894" alt="image" src="https://user-images.githubusercontent.com/3922667/151619424-6a5f2f8f-d4ea-479b-bd58-a5a418180fb9.png">

Reply:
You read my mind 😍